src/chrome/internal/stackTraces/stackTracePresentation.ts

```diff
@@ -0,0 +1,11 @@
+import { StackTracePresentationRow } from './stackTracePresentationRow';
+
+/** The stack traces we sent to the client will be represented by this classes and it is a combination of:
+ *    1. CallFrames with state information from the sync frames.
+ *    2. CodeFlowFrames without state information from async frames.
+ *    3. Labels that we use to [Show more frames] or [Frames skipped by smartStep], etc...
+ */
+ export interface StackTracePresentation {
```

Comment:
I'm confused about all the `Presentation` types. This and StackTracePresentationRow both say they are "sent to the client" but they don't really look like the DA type

Reply:
Ok. `I` name?